playbooks/ansible-test-cloud-integration-base/pre.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- hosts: controller
+  tasks:
+    - name: Setup tox role
+      include_role:
+        name: tox
+      vars:
+        tox_extra_args: "-vv -- ansible-playbook -v playbooks/ansible-test-cloud-integration-base/files/bootstrap-vcsa.yaml"
+        tox_install_siblings: false
+        zuul_work_dir: "{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
+
+    - name: Setup base virtualenv_options
+      set_fact:
+        _virtualenv_options: "--python python{{ ansible_test_python }}"
+
+    - name: Enable --system-site-packages for virtualenv_options
+      set_fact:
+        _virtualenv_options: "{{ _virtualenv_options }} --system-site-packages"
+      when: ansible_os_family == "RedHat"
+
+    - name: Create virtualenv for ansible-test
+      shell: "virtualenv {{ _virtualenv_options }} ~/venv"
+
+    - name: Install ara into virtuelenv
+      shell: ~/venv/bin/pip install "ara<1.0.0"
+
+    - name: Upgrade six for vap-runtime (workaround)
+      shell: ~/venv/bin/pip install --upgrade "six>=1.12"
+
+
+    - name: Install ansible into virtuelenv
+      shell: "~/venv/bin/pip install {{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}"
+
+    - name: Enable ARA callback plugin
```

Comment:
okay, below won't work with integration, so we likely need to modify another cfg file.

Reply:
okay, lets change to test/lib/ansible_test/_data/ansible.cfg for now

Reply:
test/integration/integration.cfg

that is what we need to create

Reply:
+1, nice catch.